docs/platform/release-notes/tizen-7-0-m2.md

```diff
@@ -0,0 +1,543 @@
+# Tizen 7.0 Public M2
+
+Release date: Oct. 31, 2022
+
+
+## Release notes
+
+### System (kernel and system framework)
+
+#### New and changed features
+
+- Kernel feature
+  - kfence, a sampling-based memory safety error detector, support with CONFIG_KFENCE on Raspberry Pi 4.
+- Resource Monitor
+  - Unified resource monitor framework and capi-system-resource-monitor have been added to monitor the various resource status.
+  - Web-based resource viewer has been added to monitor the whole resource status of a Tizen device.
+- Dlog
+  - Zero-copy buffer technique has been developed in order to reduce the printing overhead of logs.
+- Lightweight multi-user session support
+  - Lightweight multi-user session based on sub-session has been developed.
+  - Directory and file protection of users has been developed.

```

Comment:
```suggestion
  - Directory and file protection of users have been developed.

```

Reply:
Isn't "has" correct expression grammatically as the "protection" is the subject of the sentence?

Reply:
@wtprince,
Yes, you're right, '**file protection**' is the subject but so is '**directory**'. Since we have two subjects in this sentence, we suggest using '**have**' instead of '**has**'.